diff -Naur git.orig/webinterface/src/plugin.py git/webinterface/src/plugin.py
--- git.orig/webinterface/src/plugin.py	2011-11-03 10:54:30.000000000 +0800
+++ git/webinterface/src/plugin.py	2011-11-03 14:35:43.402586051 +0800
@@ -227,7 +227,7 @@
 # on given ipaddress, port, w/o auth, w/o ssl
 #===============================================================================
 def startServerInstance(session, ipaddress, port, useauth=False, l2k=None, usessl=False):
-	if hw.get_device_name().lower() != "dm7025":
+	if hw.get_device_name().lower() != "dm7025" and False:
 		l3k = None		
 		l3c = tpm.getCert(eTPM.TPMD_DT_LEVEL3_CERT)
 		
@@ -441,7 +441,7 @@
 def networkstart(reason, session):
 	l2r = False
 	l2k = None
-	if hw.get_device_name().lower() != "dm7025":		
+	if hw.get_device_name().lower() != "dm7025" and False:		
 		l2c = tpm.getCert(eTPM.TPMD_DT_LEVEL2_CERT)
 		
 		if l2c is None:
@@ -470,7 +470,7 @@
 def configCB(result, session):
 	l2r = False
 	l2k = None
-	if hw.get_device_name().lower() != "dm7025":		
+	if hw.get_device_name().lower() != "dm7025" and False:		
 		l2c = tpm.getCert(eTPM.TPMD_DT_LEVEL2_CERT)
 		
 		if l2c is None:
